scipy/stats/tests/test_censored_data.py

```diff
@@ -0,0 +1,110 @@
+# Tests for the CensoredData class.
+
+import pytest
+import numpy as np
+from numpy.testing import assert_equal
+from scipy.stats import CensoredData
+
+
+class TestCensoredData:
+
+    def test_basic(self):
+        left = [-np.inf, 1, 2, 5]
+        right = [0, 1, 3, np.inf]
+        x = [1]
+        left = [0]
+        right = [2, 5]
```

Comment:
It looks like `left` and `right` get redefined. Maybe this was left over from the old interface.

Reply:
I'll remove the first ones.